src/colors.css

```diff
@@ -7,6 +7,7 @@
   --base01:     #586e75;
   --base00:     #657b83;
   --base0:      #839496;
+  --base0-soft: rgba(131, 148, 150, 0.75);
```

Comment:
Just out of curiousness: `-soft` is not defined by solarized, it's an addition you came up with, right?